drivers/bluetooth/hci/CMakeLists.txt

```diff
@@ -36,8 +36,8 @@ if(CONFIG_DT_HAS_ST_HCI_STM32WBA_ENABLED)
   zephyr_blobs_verify(MODULE hal_stm32 REQUIRED)
 endif()
 zephyr_library_sources_ifdef(CONFIG_BT_USERCHAN    userchan.c)
-zephyr_library_sources_ifdef(CONFIG_BT_SILABS_HCI  slz_hci.c)
-if(CONFIG_DT_HAS_SILABS_BT_HCI_ENABLED)
+zephyr_library_sources_ifdef(CONFIG_BT_SILABS_EFR32 hci_silabs_efr32.c)
+if(CONFIG_DT_HAS_SILABS_BT_HCI_EFR32_ENABLED)
```

Comment:
In one side we have `BT_SILABS_EFR32` and the other side `SILABS_BT_HCI_EFR32`. I suggest to include `HCI` in all or in none of them.

Reply:
> In one side we have `BT_SILABS_EFR32` and the other side `SILABS_BT_HCI_EFR32`. I suggest to include `HCI` in all or in none of them.

The reasoning is based on context. In DTS you're far away from Bluetooth land. In `drivers/bluetooth/hci` you're deep inside it.

Reply:
Additionally, most HCI driver options don't have HCI in the name. If we add it we should add it to all drivers, and such a change is out of scope for this PR. As for DTS, "HCI" is not clear enough on its own since that concept/acronym exists in more domains than just Bluetooth.